### ISSUE TITLE
missing post-aggregator in the query editor

### ISSUE THREAD

**Issue by knoguchi** [grafana-druid-plugin/druidplugin#1]
Looks like the post-agg is missing in the query editor.   The JavaScript code to support Post Aggregation feature seems to be complete.  Maybe forgot to check in the html?

**Comment by abhisant** [comment]
Could you please elaborate ? Didn't completely follow you.. Which html are you referring to .

**Comment by knoguchi** [comment]
Your src/partials/query.editor.html has `Aggregations` section but missing `Post-Aggregations`.
@godfreyhe 's [original](https://github.com/godfreyhe/grafana-plugins/blob/master/features/druid/partials/query.editor.html#L490-L582) has to be ported.

**Comment by abhisant** [comment]
Ok. Thanks. I will have a look into it.

**Comment by knoguchi** [comment]
I [modified](https://github.com/knoguchi/druidplugin/commit/87131503a56ad8a09396319d3bd477f92d4a1664) the html.  Arithmetic type worked with a bit of fix in JS.  Quantile type didn't work but didn't investigate.

**Comment by abhisant** [comment]
ok, thanks. This repository is currently under review by Grafana team. Once that is done and the plugin is available on grafana.net, I will add the changes you requested.

**Comment by abhisant** [comment]
@knoguchi : The plugin is avaiable on grafana.net now. 
Do you want to issue a pull request of the changes that you already made. Else, I will make those changes and debug the "Quantile type" issue.

**Comment by rzarajczyk** [comment]
Hi, what is the status of this issue? Will you merge those changes?

**Comment by knoguchi** [comment]
My fork has diverged a lot for my own needs.  Query editor UI is different, resulting configuration data structure is different (not compatible with Abhi's).    So it's not mergeable.

**Comment by rzarajczyk** [comment]
thank you

**Comment by abhisant** [comment]
Sorry for the late response. I am currently traveling out of the country for 1 month on a vacation ( with intermittent internet connectivity). I will try to find some time out of my vacation to add the build scripts.

**Comment by PanYifeng** [comment]
Hi,  Can anyone tell me when could druidplugin suppert Post Aggregation feature?

**Comment by nanittu** [comment]
Is there a rc version this is merged into ? we are very interested to check this for our use case any details would help us.

**Comment by abhisant** [comment]
@knoguchi are you planning to merge this ?

**Comment by knoguchi** [comment]
Sorry, no plan.  As I mentioned in this thread, my branch has diverged significantly. 
My fork has changed the filters, aggs and post aggs to support math expression than the single field dropdown.  On top of that, I'm moving on to SQL queries for Druid 0.10.

**Comment by abhisant** [comment]
@PanYifeng I will try to add this or someone from my team will work on this and add this feature soon.

**Comment by nanittu** [comment]
@abhisant any update on the feature addition ?

**Comment by pkarmaka** [comment]
I and @nikhil2406 have just started working on this. Will get back to you on the ETA soon.

**Comment by abhisant** [comment]
Thanks @pkarmaka  and @nikhil2406.

**Comment by nanittu** [comment]
@pkarmaka any ETAs on this ? Sorry for the push. we are on the brink of making a time sensitive product decision.

**Comment by pkarmaka** [comment]
@nanittu  ETA is around 25th-29th June..

**Comment by nikhil2406** [comment]
@nanittu we have completed this change and currently in process of verifying it. We'll sending out pull request soon.

**Comment by abhisant** [comment]
Thank you @nikhil2406  @pkarmaka for quick turnaround.

**Comment by nanittu** [comment]
@abhisant @pkarmaka @nikhil2406 Thank you ! appreciate your timely response.

**Comment by abhisant** [comment]
@nanittu : @nikhil2406 and @pkarmaka issued a pull, it was reviewed / merged and tested. The feature is present in the repo, you can copy the dist contents and update your plugin. I will release it soon as a part of Grafana plugin. 
Please let us know if you face any issues.
